content/en/getting-started/configuration.md

```diff
@@ -386,6 +406,10 @@ See [configure page](#configure-page).
 
 See [configure pagination](/templates/pagination/#configuration).
 
+###### panicOnWarning
+
+(`bool`) Panic on first WARNING log. Default is `false`.
```

Comment:
```suggestion
(`bool`) Whether to panic on first the WARNING log. Default is `false`.
```

Reply:
Suggestions:
1. Whether to panic on first WARNING. 
2. Whether to panic on the first WARNING.

Reply:
What about that last one? I would use:
>Whether to panic on first WARNING.